teammapper-frontend/mmp/src/map/handlers/nodes.ts

```diff
@@ -806,6 +810,7 @@ export default class Nodes {
 
             if (graphic === false) {
                 node.colors.name = sanitizedColor
+                return true
```

Comment:
if graphic is true and `node.colors.name !== color`, then the return value is still undefined.

Reply:
Good point, I've moved the `return true` outside of the `if (graphic)` since the node has already been updated anyway.